flash/core/data/data_source.py

```diff
@@ -697,3 +709,218 @@ def _get_classes(self, data):
             classes = data.distinct(label_path)
 
         return classes
+
+
+class LabelStudioDataSource(DataSource):
```

Comment:
Let's move this to its own file under `flash/core/integrations/labelstudio/data_source.py`

Reply:
Done